interleave.py

```diff
@@ -0,0 +1,24 @@
+# import the cycle and islice functions from the itertools module
+from itertools import cycle, islice
+
+
+# define a function called round_robin that accepts any number of iterables
+def round_robin(*iterables):

```

Comment:
The comments make the code a little bit messy, I would use documentation instead https://realpython.com/documenting-python-code/#commenting-vs-documenting-code